tests/quotes/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -64,6 +64,10 @@ exports[`strings.js 1`] = `
 // Unnecessary escapes.
 "\\'"
 '\\"'
+"\\a"
+'\\a'
+"hol\\a"
+'hol\\a'
```

Comment:
Yeah, this definitely needs to be thoroughly tested. I added some more tests in https://github.com/prettier/prettier/pull/1575/commits/f04b200121c1986a750102f8b7342e46754d9994, let me know if you spot any holes.